src/main/java/gov/nih/nci/evs/api/aop/MetricAdvice.java

```diff
@@ -41,6 +56,28 @@
   @Autowired
   ElasticOperationsService operationsService;
 
+  boolean databaseFound;
+
+  @PostConstruct
+  public void postInit() throws Exception {
+    File file = new File(applicationProperties.getMetricsDir() + "/GeoLite2-City.mmdb");
+    this.databaseFound = file.exists();
+    if (databaseFound) {
+      dbReader = new DatabaseReader.Builder(file).build();
+    } else {
+      logger.warn("GeoLite Database was not found = " + applicationProperties.getMetricsDir());
+    }
+
+    String indexName = "metrics-" + String.valueOf(Calendar.getInstance().get(Calendar.YEAR)) + "-"
+        + String.valueOf(Calendar.getInstance().get(Calendar.MONTH));
```

Comment:
Do me a favor here and make the month part be +1 so Jan=1, Feb=2.  The "zero indexed" thing will get annoying so let's fix it now.